builds/e2e/templates/e2e-setup.yaml

```diff
@@ -19,69 +19,48 @@ steps:
-      Write-Output '##vso[task.setvariable variable=whichPackageBuild]latestFromBranch'
-      Write-Output '##vso[task.setvariable variable=whichImageBuild]specific'
-      Write-Output '##vso[task.setvariable variable=imageBuildId]$(Build.TriggeredBy.BuildId)'
-    }
-    # 3) Manual build (use given args for 'Build Images' build ID and 'Edgelet Packages' build ID)
-    elseif ('$(az.pipeline.packages.buildId)' -and '$(az.pipeline.images.buildId)')
+    $imageBuildId = $(resources.pipeline.images.runID)
+    $packageBuildId = $(resources.pipeline.packages.runID)
+
+    if ('$(az.pipeline.images.buildId)')
     {
-      Write-Output '##vso[task.setvariable variable=whichPackageBuild]specific'
-      Write-Output '##vso[task.setvariable variable=whichImageBuild]specific'
-      Write-Output '##vso[task.setvariable variable=packageBuildId]$(az.pipeline.packages.buildId)'
-      Write-Output '##vso[task.setvariable variable=imageBuildId]$(az.pipeline.images.buildId)'
+      Write-Output '>> User supplied az.pipeline.images.buildId=$(az.pipeline.images.buildId)'
+      $imageBuildId = '$(az.pipeline.images.buildId)'
     }
-    else # Error
+
+    if ('$(az.pipeline.packages.buildId)')
     {
-      if ($env:BUILD_TRIGGEREDBY_DEFINITIONNAME)
-      {
-        Write-Output "Build triggered by unrecognized build '$env:BUILD_TRIGGEREDBY_DEFINITIONNAME'"
-      }
-      else
-      {
-        Write-Output "Missing build parameter 'az.pipeline.packages.buildId' or 'az.pipeline.images.buildId'"
-      }
-      exit 1
+      Write-Output '>> User supplied az.pipeline.packages.buildId=$(az.pipeline.packages.buildId)'
+      $packageBuildId = '$(az.pipeline.packages.buildId)'
     }
-  displayName: Locate edgelet packages, runtime images
 
-- task: DownloadBuildArtifacts@0
-  displayName: Download edgelet packages
-  inputs:
-    buildType: specific
-    project: one
-    pipeline: $(az.pipeline.packages)
-    buildVersionToDownload: $(whichPackageBuild)
-    branchName: refs/heads/master
-    buildId: $(packageBuildId)
-    downloadType: specific
-    itemPattern: $(artifactDir)/*
+    Write-Output "##vso[task.setvariable variable=imageBuildId]$imageBuildId"
+    Write-Output "##vso[task.setvariable variable=packageBuildId]$packageBuildId"
+  displayName: Override artifacts with user-supplied args
 
 - task: DownloadBuildArtifacts@0
   displayName: Get Docker image info
   inputs:
     buildType: specific
-    project: one
-    pipeline: $(az.pipeline.images)
-    buildVersionToDownload: $(whichImageBuild)
-    branchName: refs/heads/master
+    project: $(resources.pipeline.images.projectID)
+    pipeline: $(resources.pipeline.images.pipelineName)
+    buildVersionToDownload: specific
     buildId: $(imageBuildId)
-    downloadType: specific
+    downloadType: single
+    artifactName: $(az.pipeline.images.artifacts)
     itemPattern: $(az.pipeline.images.artifacts)/artifactInfo.txt
```

Comment:
nit:can "$(az.pipeline.images.artifacts)" be removed?

Reply:
If I removed it, I'd have to replace it with the hard-coded value `core-linux`. Is that what you're suggesting?